Header header
int32 frame_id
geometry_msgs/Pose pose
geometry_msgs/Point32 cov
geometry_msgs/Point32 posi_vis
float32 cov_vis
float32[] us
float32[] vs
geometry_msgs/Point32[] mps
int32[] vecInlierIdx
